dockers/docker-dhcp-server/cli/config/plugins/dhcp_server.py

```diff
@@ -1,13 +1,371 @@
+    ipaddress.ip_address(ip_end)
+    dbconn = db.db
+    key = "DHCP_SERVER_IPV4_RANGE|" + range_name
+    db.hmset("CONFIG_DB", key, {"ranges": ip_start + "," + ip_end})
+
+
+@range.command()
+@click.argument("range_name", required=True)
+@clicommon.pass_db
+def del(db, range_name)
+    dbconn = db.db
+    key = "DHCP_SERVER_IPV4_RANGE|" + range_name
+    db.delete("CONFIG_DB", key)
+
+
+@ipv6.group(cls=clicommon.AliasedGroup)
+def range():
+    pass
+
+
+@range.command()
+@click.argument("range_name", required=True)
+@click.argument("ip_start", required=True)
+@click.argument("ip_end", required=False)
+@clicommon.pass_db
+def add(db, range_name, ip_start, ip_end):
+    pass
+
+
+@range.command()
+@click.argument("range_name", required=True)
+@click.argument("ip_start", required=True)
+@click.argument("ip_end", required=False)
+@clicommon.pass_db
+def update(db, range_name, ip_start, ip_end):
+    pass
+
+
+@range.command()
+@click.argument("range_name", required=True)
+@clicommon.pass_db
+def del(db, range_name)
+    pass
+
+
+@ipv4.group(cls=clicommon.AliasedGroup)
+def ip():
+    pass
+
+
+@ip.command()
+@click.argument("vlan_interface", required=True)
+@click.argument("interface", required=True)
+@click.options("--range", required=False, nargs=-1)
+@click.argument("ip_list", required=False, nargs=-1)
+@clicommon.pass_db
+def bind(db, vlan_interface, interface, range_, ip_list):
+    dbconn = db.db
+    key = "DHCP_SERVER_IPV4_PORT|" + vlan_interface + "|" + interface
+    if range_:
+        dbconn.hmset("CONFIG_DB", key, {"ranges": ",".join(range_)})
+    elif ip_list:
+        dbconn.hmset("CONFIG_DB", key, {"ips": ",".join(ip_list)})
+    else:
+        raise Exception("One of range and ip_list should be provided")
+
+
+@ip.command()
+@click.argument("vlan_interface", required=True)
+@click.argument("interface", required=True)
+@click.options("--range", required=False, nargs=-1)
+@click.argument("ip_list", required=False, nargs=-1)
+@clicommon.pass_db
+def unbind(db, vlan_interface, interface, range_, ip_list):
+    pass
+
+
+@ipv6.group(cls=clicommon.AliasedGroup)
+def ip():
+    pass
+
+
+@ip.command()
+@click.argument("vlan_interface", required=True)
+@click.argument("interface", required=True)
+@click.options("--range", required=False, nargs=-1)
+@click.argument("ip_list", required=False, nargs=-1)
+@clicommon.pass_db
+def bind(db, vlan_interface, interface, range_, ip_list):
+    pass
+
+
+@ip.command()
+@click.argument("vlan_interface", required=True)
+@click.argument("interface", required=True)
+@click.options("--range", required=False, nargs=-1)
+@click.argument("ip_list", required=False, nargs=-1)
+@clicommon.pass_db
+def unbind(db, vlan_interface, interface, range_, ip_list):
+    pass
+
+
+@ipv4.group(cls=clicommon.AliasedGroup)
+def option():
+    pass
+
+
+@option.command()
+@click.argument("option_name", required=True)
+@click.argument("option_id", required=True)
+@click.argument("type", required=True)
```

Comment:
Suggest changing to no required and default "string" for feature support assigned options which type should not be customized and should follow ietf doc